packages/core/src/compiler.ts

```diff
@@ -91,9 +101,10 @@ export class Compiler<Identity extends Saved | Unsaved = Saved> {
       modules: outputModules,
       deps: [...this.dependencies],
     };
+    return this.resolveFieldsWithPlaceholders(compiledCard);
```

Comment:
@ef4 i don't think I can move this function to being one that acts only on the fields, as the nature of replacing the placeholders is that I need a handle on the compiled card that is being assembled since it is the thing that is replacing the placeholders